src/modules/RokulRunnings.ts

```diff
@@ -717,8 +756,14 @@ export class RokulRunnings {
   private squashAttributes(responseObject: AppUIResponseObject[]) {
     const elementsArray: SquashedAppUIObject[] = [];
     responseObject.forEach(element => {
-      const elementName = Object.keys(element)[0];
+      let elementName = Object.keys(element)[0];
       const childElement = element[elementName] as AppUIResponseObject;
+      const childElementAttributes = childElement.attributes;
+      if (childElementAttributes.name) {
+        childElementAttributes.tag = elementName;
+        elementName = childElementAttributes.name;
+        delete childElementAttributes.name;
```

Comment:
I think it's best to leave `name` and `tag` as attributes no matter what. Makes it much easier to access both, instead of finding a way to access the key of the element

Reply:
so...
```
{
    Label: {
        name: "Element"
    }
}
```
becomes
```
{
    Element: {
        name: "Element",
        tag: "Label"
    }
}
```
?

Reply:
yes pls

Reply:
Should 
```
{
    Label: {
        color: "#12345612"
    }
}
```
become
```
{
    Label: {
        color: "#12345612",
        tag: "Label"
    }
}
```
?

Reply:
yeah would be nice